src/graphics/texture.js

```diff
@@ -607,7 +607,7 @@ class Texture {
 
                 if (format === PIXELFORMAT_PVRTC_2BPP_RGB_1 ||
                     format === PIXELFORMAT_PVRTC_2BPP_RGBA_1) {
-                    blockWidth = Math.floor(blockWidth / 2, 1);
+                    blockWidth = Math.floor(blockWidth / 2);
```

Comment:
@slimbuck This looks super-odd to me. I wonder what the original intention of the section parameter of 1 to `Math.floor` was here. Can you cast an eye and verify the logic is OK?

Reply:
Bug! I think this should be:
```javascript
blockWidth = Math.max(Math.floor(blockWidth / 2), 1);
```

Reply:
Thanks. Fixed!